packages/components/src/internal/components/samples/constants.ts

```diff
@@ -277,6 +277,7 @@ export const SAMPLE_DOMAIN_DEFAULT_SYSTEM_FIELDS = [
         Required: false,
         Description: '',
         Disableble: true,
+        Editable: false,
```

Comment:
minor: seems a bit odd that we have an optional boolean prop where undefined is equivalent to true. Should we make this a required prop and explicitly have each field set this to true/false for clarity?